### PR TITLE
oracel-api-fix-changes

### PR DESCRIPTION
**Pull request by harjaapdhillon16**
Added UI for handling when the oracle api doesn't return anything

### CODE REVIEW

**Comment by harjaapdhillon16** [comment]
Screenshot 2023-04-27 at 23 28 42

File: src/components/pages/landing/Gooddollar.js [hunk 1]
```diff
@@ -29,102 +29,99 @@ export const Gooddollar = ({ setShowGooddollarVerification }) => {
...
 
-      try {
-        const result = await verifyUser(sendObj);
-        const { data } = await supabase.select('users', {
+        const sendObj = {
+          m: JSON.stringify(rawData),
+          c: wallet.accountId,
+          sig: rawGoodDollarData.sig,
+        };
+        console.log(sendObj);
```
Comment (Megha-Dev-19) [inline src/components/pages/landing/Gooddollar.js]: do we need it?

File: src/components/pages/landing/Gooddollar.js [hunk 2]
```diff
@@ -29,102 +29,99 @@ export const Gooddollar = ({ setShowGooddollarVerification }) => {
...
+              updateData,
+              { wallet_identifier: wallet.accountId }
+            );
+          } else {
+            const { error: appError } = await supabase.insert(
+              'users',
+              updateData
+            );
+          }
+          console.log(result);
```
Comment (Megha-Dev-19) [inline src/components/pages/landing/Gooddollar.js]: same

File: src/components/pages/landing/Gooddollar.js [hunk 3]
```diff
@@ -29,102 +29,99 @@ export const Gooddollar = ({ setShowGooddollarVerification }) => {
...
+      validate: (values) => {
+        const errors = {};
+        // if (values.phone) {
+        //   if (!isPossiblePhoneNumber(values.phone)) {
+        //     errors.phone =
+        //       "Invalid phone number, please provide the phone number with valid country code";
+        //   }
+        // } else {
+        //   errors.phone = "Phone Number Required";
+        // }
```
Comment (Megha-Dev-19) [inline src/components/pages/landing/Gooddollar.js:118]: can you add the reason about why we have commented it

**Review by Megha-Dev-19 — COMMENTED** [review]
Could you provide brief summary about the changes you did (like how you solved the oracle issue)? Please do it in every PR

**Review by Megha-Dev-19 — APPROVED** [review]

File: src/components/pages/landing/Gooddollar.js (hunk 3 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/Gooddollar.js:118]: We are not using the user phone number any more hence validation not needed

File: src/components/pages/landing/Gooddollar.js (hunk 2 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/Gooddollar.js]: I'll remove the console log

File: src/components/pages/landing/Gooddollar.js (hunk 1 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/Gooddollar.js]: No I'll remove it

**Comment by harjaapdhillon16** [comment]
Sure will do

**Pull request merged**